### PR TITLE
ref(ts) Convert IndicatorStore to typescript

### PR DESCRIPTION
**Pull request by markstory**

### CODE REVIEW

**Review by evanpurkhiser — APPROVED** [review]

File: src/sentry/static/sentry/app/stores/indicatorStore.tsx [hunk 1]
```diff
@@ -1,9 +1,66 @@
...
+  /**
+   * Alias for add()
+   */
+  addMessage: (
+    message: string,
+    type: Indicator['type'],
+    options?: Indicator['options']
+  ) => Indicator;
+
+  /**
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/stores/indicatorStore.tsx:45]: Minor inconsistency on new lines between methods

File: src/sentry/static/sentry/app/stores/indicatorStore.tsx [hunk 2]
```diff
@@ -1,9 +1,66 @@
...
+  ) => Indicator;
+  /**
+   * Alias for add()
+   */
+  addMessage: (
+    message: string,
+    type: Indicator['type'],
+    options?: Indicator['options']
+  ) => Indicator;
+
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/stores/indicatorStore.tsx:44]:
```suggestion
```

File: src/sentry/static/sentry/app/stores/indicatorStore.tsx [hunk 3]
```diff
@@ -1,9 +1,66 @@
...
+  /**
+   * Remove all current indicators.
+   */
+  clear: () => void;
+
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/stores/indicatorStore.tsx:49]:
```suggestion
```

**Review by priscilawebdev — APPROVED** [review]
it looks good! :)

**Pull request merged**